chart/templates/rbac/pod-launcher-role.yaml

```diff
@@ -19,7 +19,11 @@
 ## Airflow Pod Launcher Role
 #################################
 {{- if and .Values.rbacEnabled .Values.allowPodLaunching }}
+{{- if .Values.multiNamespaceMode }}
 kind: ClusterRole
+{{- else }}
+kind: Role
```

Comment:
> when you create a Role, you have to specify the namespace it belongs in.

> ClusterRole, by contrast, is a non-namespaced resource.

https://kubernetes.io/docs/reference/access-authn-authz/rbac/

Can you add namespace?

Reply:
@mik-laj fixed 👍